src/elements/content-sidebar/activity-feed/common/activity-message/ActivityMessage.scss

```diff
@@ -13,18 +13,49 @@
     }
 }
 
+.bcs-ActivityMessage-edited {
+    color: $bdl-gray-62;
+}
+
 .bcs-ActivityMessage-mention {
     overflow: initial;
     white-space: normal;
 }
 
-.bcs-ActivityMessage-translate {
+.bcs-ActivityMessage-collapsed {
+    /* webkit-line-clamp only works in combination with -webkit-box */
+    /* stylelint-disable-next-line value-no-vendor-prefix */
+    display: -webkit-box;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    overflow-wrap: break-word;
+    -webkit-line-clamp: 7;
+    -webkit-box-orient: vertical;
+}
+
+.btn-plain.bcs-ActivityMessage-translate,
```

Comment:
Minor, but this can be simplified:

```
.bcs-ActivityMessage-toggleMoreLess,
.bcs-ActivityMessage-translate {
    &.btn-plain {
        // ... styles
    }
}

Reply:
I didn't knew exact syntax for that, but I knew you'll tell me 😉